dbt_project.yml

```
@@ -1278,4 +1278,9 @@ models:
         polygon:
           +schema: account_abstraction_erc4337_polygon
         gnosis:
-          +schema: account_abstraction_erc4337_gnosis
\ No newline at end of file
+          +schema: account_abstraction_erc4337_gnosis
+
+    ironbank_v2:
+      +schema: ironbank_v2
+      goerli:
+        +schema: ironbank_v2_goerli
```

nit: new line at EOF.
```suggestion
        +schema: ironbank_v2_goerli

```